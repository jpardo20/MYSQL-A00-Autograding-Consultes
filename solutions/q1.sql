-- TODO: SELECT a.nom FROM ... JOIN ... WHERE c.nom = '1r de DAM' ORDER BY a.nom ASC;
