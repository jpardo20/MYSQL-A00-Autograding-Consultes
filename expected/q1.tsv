Bases de Dades I
Llenguatges de Marques
Programaci√≥ I
